lib/modules/dosomething/dosomething_reportback/dosomething_reportback.query.inc

```diff
@@ -130,7 +130,7 @@ function dosomething_reportback_get_reportbacks_query($params) {
  * @param  array  $params
  * @return mixed
  */
-function dosomething_reportback_filter_flagged_reportbacks($params) {
+function dosomething_reportback_filter_flagged_reportbacks($params, $load_all) {
```

Comment:
good call about the naming - I can definitely change to `include_flagged` for clarity! 

The user won't actually be returned the flagged results, it's more so just checking to see if there are flagged items or not and if so, then qualifies for a `403` status code.